plugins/module_utils/vmware.py

```diff
@@ -1140,6 +1140,19 @@ def is_vcenter(self):
         elif api_type == 'HostAgent':
             return False
 
+    def vcenter_version_at_least(self, version=None):
+        """
+        Check that the vCenter server is at least a specific version number
+        Args:
+            version (tuple): a version tuple, for example (6, 7, 0)
+        Returns: bool
+        """
+        if version:
+            vc_version = self.content.about.version
+            return StrictVersion(vc_version) >= StrictVersion('.'.join(map(str, version)))
+        else:
+            self.module.fail_json(msg='The passed vCenter version: %s is None.' % version)
```

Comment:
```suggestion
        if version:
            vc_version = self.content.about.version
            return StrictVersion(vc_version) >= StrictVersion('.'.join(map(str, version)))
        self.module.fail_json(msg='The passed vCenter version: %s is None.' % version)
```

 No need for `else` after using `return`

Reply:
Yeah, thanks for reviewing.